boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -161,17 +195,26 @@ document.addEventListener("DOMContentLoaded", () => {
     try {
       const response = await fetch(apiUrl);
       const ssrDomEventString = await response.text();
-
       onLoadMoreButtonBehaviour(ssrDomEventString);
       const newEvents = parser.parseFromString(ssrDomEventString, "text/html");
-
       eventsContainer.append(...newEvents.body.children);
+      initModals(eventsContainer.querySelectorAll(".event"));
       goTo(`/search?${paramsAsString}`);
     } catch (e) {
       console.error(e);
     }
   };
 
+  const initModals = (events) => {
+    events.forEach(event => {
+      const anchor = event.querySelector(".anchor-to-event");
+      const content = event.querySelector("#modal-content");
+      anchor.addEventListener("click", () => {
```

Comment:
i am pretty sure you add this event listener to already existing events as well, so events can get multiple event listeners when you click "load more"